pyproject.toml

```diff
@@ -55,7 +55,8 @@ explicit = true
 
 # flash-attn related setups
 [project.optional-dependencies]
-compile = ["flash-attn>=2.8.0.post1"]
+compile = ["flash-attn>=2.8.0.post1",
```

Comment:
I know its not exactly related but can we fix the flash-attn to be post2 like it is in the dockerfile

Reply:
Yeah I can do that!